scripts/windows/VERIFICATION.txt

```diff
@@ -0,0 +1,17 @@
+VERIFICATION
+Verification is intended to assist the Chocolatey moderators and community
+in verifying that this package's contents are trustworthy.
+ 
+To verify logdna-agent.exe:
+ 
+1. Download <https://s3.amazonaws.com/repo.logdna.com/win/logdna-agent.latest.zip> which includes "logdna-agent.exe"
+2. You can use one of the following methods to obtain the checksum
```

Comment:
Sorry one more, I think "calculate the checksum" is slightly better than "obtain the checksum".

Reply:
You are right! Done!